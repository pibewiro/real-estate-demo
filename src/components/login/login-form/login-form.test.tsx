import { render } from '@testing-library/react';
import LoginForm from '.';

describe('House List Component', () => {
  it('Should Render House list Component', () => {
    const Component = render(
      <LoginForm
        email="email"
        setEmail={() => console.log()}
        password="password"
        setPassword={() => console.log()}
        handleLogin={() => console.log()}
      />
    );
    expect(Component.container.firstChild).toMatchSnapshot();
  });
});
