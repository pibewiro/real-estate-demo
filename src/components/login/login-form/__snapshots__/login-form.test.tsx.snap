// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`House List Component Should Render House list Component 1`] = `
<div
  class="login"
>
  <div
    class="login__logo"
  >
    <span
      style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
    >
      <img
        alt="login image"
        data-nimg="fill"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
      />
      <noscript />
    </span>
  </div>
  <input
    class="input"
    placeholder="Email"
    type="text"
    value="email"
  />
  <input
    class="input"
    placeholder="Password"
    type="text"
    value="password"
  />
  <button
    style="width: 100%;"
  >
    Login
  </button>
</div>
`;
